pyproject.toml

```diff
@@ -11,17 +11,18 @@ packages = [{include = "extremeflash"}]
 python = "^3.10"
 pyserial = "^3.5"
 tftpy = "^0.8.2"
-paramiko = "^3.0.0"
+paramiko = "^3.3.1"
 scp = "^0.14.5"
+poetry = "^1.7.1"
```

Comment:
I don't think we want to install poetry inside the poetry venv 😉

Reply:
fixed poetry inside the poetry venv
downgraded paramico as requested